lib/generic/center.h

```diff
@@ -8,20 +8,29 @@
 #pragma once
 
 #include "cluster_descriptor.h"
+#include "generic/point_cloud.h"
 
 namespace MouseTrack {
 
-
 class Center : public ClusterDescriptor {
 public:
-	/// Returns a distance/norm `n` between `this` and `other`.
-	/// It should be small for two instances that are "similar" to each other,
-	/// and large for two instances that are very different.
-	/// Important properties:
-	/// 1. 0 <= n <= inf
-	/// 2. if this == other -> n == 0
-	virtual double compare(const ClusterDescriptor* other) const;
-};
+  Center(Coordinate x, Coordinate y, Coordinate z) : _x(0), _y(0), _z(0) {}
```

Comment:
You probably wanted: `_x(x), _y(y), _z(z)`

Second: I know it's nearly nothing, but I also would push the Constructor and Destructor implementation to the .cpp file and only leave the declaration here.

Reply:
What's the benefit? Just style?